internal/cephfs/clone.go

```diff
@@ -134,7 +134,11 @@ func createCloneFromSubvolume(ctx context.Context, volID, cloneID volumeID, volO
 	return nil
 }
 
-func cleanupCloneFromSubvolumeSnapshot(ctx context.Context, volID, cloneID volumeID, parentVolOpt *volumeOptions) error {
+func cleanupCloneFromSubvolumeSnapshot(
+	ctx context.Context,
+	volID, cloneID volumeID,
+	parentVolOpt *volumeOptions,
```

Comment:
is the comma at the end really required? it looks so incomplete when used with function calls